src/patternfly/components/Table/table.scss

```diff
@@ -319,8 +326,10 @@
 
 // Table action cell
 // ==================================================================
-.pf-c-table__action {
+.pf-c-table__action,
+.pf-c-table__inline-edit-action {
```

Comment:
should this be a modifier of `.pf-c-table__action`?

Reply:
Yes, I think it's cleanest to separate the two entirely. @seanforyou23 wdyt?